### ISSUE TITLE
XCSoar occasionally says: OpenVario on TCP port 4352 not connected

### ISSUE THREAD

**Issue by phorward** [Openvario/meta-openvario#136]
Hello!

I've downloaded the latest 20193 release for my OpenVario with 5,7" display and now recognize a strange behavior: The OpenVario device occasionally is found in XCSoar and runs immediately and with vario sound out of the box, but in most times when I start the computer, it states "Not connected" as you can seen from the photo here:
![photo5877657762870180336](https://user-images.githubusercontent.com/16870072/120076898-230c9d00-c0a8-11eb-9b5c-89f13a63d464.jpg)

It seems to be a race condition or some kind, and I'm curious that nobody else every had this problem before. 
Is there any way to fix it?

**Comment by sfalmo** [comment]
What sensorboard do you use? I have seen this problem only once about two weeks ago with a SteFly 5.7'' running on 20193 stable. But I did not think much of it since it did not occur again in the numerous times I have used it.

This could be a pain to debug, but if you can reproduce it frequently, maybe take a look at some of the logs.
I personally don't think it is XCSoars fault, but you can still check `xcsoar.log`.
Instead, I suspect sensord to be the problem here; its error messages should hopefully appear in `journalctl -u sensord`.

**Comment by phorward** [comment]
Hello @sfalmo, thanks for your reply.

Well, I think it's gonna be a huger problem with my hardware. Its not a SteFly, I ordered it used somewhere. Two weeks ago, when it was very warm outside, my OpenVario froze like shown below entirely for about 5 times during flight. I restarted it several times and it froze again occasionally after two minutes or so. I think there is something generall wrong with it, and I want to replace it now with a SteFly one.

![photo5997002813693080958](https://user-images.githubusercontent.com/16870072/124922284-a3a0b080-dff9-11eb-81ae-77534ec12a5f.jpg)

I think this is not a problem with OpenVario. I'll check the logs and so some more tests when I replaced it with the new one soon.
